velox/docs/monthly-updates/november-2023.rst

```diff
@@ -0,0 +1,96 @@
+********************
+November 2023 Update
+********************
+
+Documentation
+=============
+* Add documentation for ``num_cached_file_handles`` :doc:`configuration property</configs>`.
+
+
+Core Library
+============
+* Add support to export constant vector as Arrow REE array. :pr:`7327`, :pr:`7398`
+* Add `libcpr` a lightweight http client. :pr:`7385`
```

Comment:
maybe turn libcpr into a link?

let's move to Performance and Correctness (this is testing only)